rero_ils/modules/selfcheck/api.py

```diff
@@ -427,3 +429,70 @@ def selfcheck_checkin(transaction_user_pid, item_barcode, **kwargs):
                 _('Error encountered: please contact a librarian'))
             raise SelfcheckCirculationError('self checkin failed', checkin)
         return checkin
+
+
+def selfcheck_renew(transaction_user_pid, item_barcode, **kwargs):
+    """SIP2 handler to perform renew.
+
+    perform renew action received from the selfcheck.
```

Comment:
```suggestion
    Perform renew action received from the selfcheck.
```
I'm not sure about it, but I would say it's a sentence.

Reply:
done